src/components/Select/Select_shim.css

```
@@ -89,6 +121,22 @@ div.neo-multiselect.neo-multiselect--small span.neo-multiselect-combo__header {
   padding-bottom: 0;
 }
 
+.neo-multiselect
+  span.neo-multiselect-combo__header
+  button.neo-multiselect__header.neo-button--width-10px {
```

Suggest an alternate implementation in order to avoid overly-verbose class names 

```suggestion
  button.neo-multiselect__header:last-child {
```

---

great